src/test/ui/rfc-2497-if-let-chains/feature-gate.rs

```diff
@@ -1,4 +1,4 @@
-// gate-test-let_chains
+// check-pass
```

Comment:
I have the impression that typically we just delete feature-gate tests when a feature is stabilized, as long as the actual functionality is tested in other tests.

Reply:
Renamed to `allowed-syntax.rs` as the file's contents are useful to preserve the robustness of the parser. Nevertheless, it can be removed if desired.